### DISCUSSION TITLE
CSS for a custom circular menu.

### DISCUSSION THREAD

**Discussion by kontosa** [Pitmairen/selection-search#132] [Show and tell]
Been messing around with CSS (poorly) to make a circular menu, may be useful for others to start with and improve. 
Looks like this:

![image](https://user-images.githubusercontent.com/102735221/214758701-6a92d68d-2e49-4fe0-b6fa-80f6b1e983c7.png)

Works like:

![vivaldi_KVpPFQMjJl](https://user-images.githubusercontent.com/102735221/214756870-93c6f1bd-ba38-4403-bd60-c859c1acf207.gif)

Once text is selected it opens and stays open until you move outside of the ring (meaning you can click multiple items to open in background tabs in one go). Use Advanced settings to control this and others (see below). It will format your existing items around the ring, so you don't lose anything.

CSS code, paste this into the Advanced Settings/Customise button/text area:
````
.popup img {
  height: 20px;
  width: 20px;
}
.popup{
 border-width: 40px;
 box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.29);
}
.mainmenu {
 border-color: rgba(255, 255, 255, 0.9) /* 45, 147, 237 with opacity */

}
 pointer-events: none;
}
.mainmenu *{
 pointer-events: auto;

}
.submenu{
  border-color: #444;
}
.popup img{
 -webkit-filter: grayscale(20%);
 transition: -webkit-filter 0.2s ease-in-out;
 /*box-shadow: 2px 3px 6px 0px rgba(0, 0, 0, .15);*/
}
.popup a:hover img, .popup a.active img{
 -webkit-filter: grayscale(0%);
 transform: scale(1.2);

}
.popup a:hover{

}
.popup .engine-separator{
 background: ;
}
````

The preview may look weird due to its spacing, but test on real sites after refreshing them.
My settings for this:
![image](https://user-images.githubusercontent.com/102735221/214757500-17ff5539-f2fd-4ba4-a831-e0a59e0cc1e4.png)

**Comment by Pitmairen** [comment]
Looks good! If someone wants to try it, it is important to mention that it requires the experimental circular menu option to be checked:
![image](https://user-images.githubusercontent.com/357032/214874915-64451497-c91d-454f-b925-77a2c2ba35f7.png)
